app/views/finance/balancing/_group_order_articles.html.haml

```diff
@@ -10,7 +10,7 @@
         %td= t('.total')
         %td{:colspan => "3",:style => "width:14em"}
           = link_to t('.add_group'), new_group_order_article_path(order_article_id: order_article.id),
-            remote: true, class: 'btn btn-mini'
+            remote: true, class: 'btn btn-mini' unless order_article.order.closed?
```

Comment:
Since this is all the way at the end of a long line, I'd put the conditional on a separate line:
```haml
%td{:colspan => "3",:style => "width:14em"}
  - unless order_article.order.closed?
    = link_to t('.add_group'), new_group_order_article_path(order_article_id: order_article.id), remote: true, class: 'btn btn-mini'
```

Reply:
i got busy and missed this.  i agree.  this is addressed by latest commit (hopefully it is ok, because i used the github online editor)